src/app/ToDo/Components/to-do.component.ts

```diff
@@ -1,66 +1,92 @@
 import { Component, OnInit } from '@angular/core';
+import {
+  TodosState
+} from '../state'
+import * as TodoActions from '../state/todo.action'
+import { ToDo } from '../state/todo.model';
+import { Store } from '@ngrx/store';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ToDoFacade } from '../state/todo.facade';
 import { Observable, Subscription } from 'rxjs';
-import { map } from 'rxjs/operators';
-import ToDo from '../state/todo.model';
-import ToDoState from '../state/todo.state';
-import { ToDoFacade } from './../state/todo.facade';
 
 @Component({
   selector: 'app-to-do',
-  templateUrl: './to-do.component.html',
-  styleUrls: ['./to-do.component.css']
+  templateUrl: './to-do.component.html'
 })
 export class ToDoComponent implements OnInit {
-  todo$: Observable<ToDoState>;
-  ToDoSubscription: Subscription;
-  ToDoList: ToDo[] = [];
 
-  Title: string = '';
-  IsCompleted: boolean = false;
-  todoError: Error = null;
+  public todo$: Observable<any>;
+  public todoForm: FormGroup;
+  public toDoSubscription: Subscription;
 
-  constructor(private _todoFacade: ToDoFacade) {
+  public loadTask(task: ToDo) {
+    this.todoForm.patchValue(task);
```

Comment:
Here Angular is inconsistent because what you wrote is ok, but it can happen that people report you missing a modifier (public, private) or return type. I understand perfectly well that it is public and the return type is automatically detected by TS. I personally add the return type each time.

Reply:
Are you saying about the return type of the loadTask method?

Reply:
Yes.